.eslintrc.js

```diff
@@ -198,7 +198,15 @@ module.exports = {
     'use-isnan': 'error',
     'valid-typeof': 'error',
   },
-  ignorePatterns: ['dist', 'tsconfig.json', 'tsconfig.eslint.json', 'package.json', 'package-lock.json', 'ort'],
+  ignorePatterns: [
+    'dist',
+    'tsconfig.json',
+    'tsconfig.eslint.json',
+    'package.json',
+    'package-lock.json',
+    'renovate.json',
+    'ort',
+  ],
```

Comment:
can we configure renovate such that it does not create a PR per change but groups its updates? Maybe the following groups would make sense:
* all prod dependencies
* all eslint stuff
* all types
* other dev dependencies
?

Reply:
Sounds good!

Reply:
I'm just not sure about types - I'd leave that group out - it would feel weird putting all the types in their own category since they're not really related to each other. What do you think?

Reply:
Actually on second thought it probably makes perfect sense.